packages/cli/internal/pkg/cli/format/format.go

```diff
@@ -4,24 +4,28 @@ import (
 	"os"
 	"text/tabwriter"
 )
+var Default Formatter
 
-var Formats = map[string]Formatter{
-	"text":    textWriter,
-	"tabular": tabularWriter,
-}
-var Format string
+type FormatterType string
+const (
+	textFormat FormatterType = "text"
+	tabularFormat FormatterType = "tabular"
```

Comment:
I think the string value for this should be just "table".

For more details see aws cli output options:
- https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-output-format.html

I would suggest to follow the same names, to avoid confusion.